src/tools/runk/libcontainer/src/container.rs

```diff
@@ -144,9 +146,9 @@ impl Container {
                 .hooks
                 .as_ref()
                 .ok_or_else(|| anyhow!("hooks config was not present"))?;
-            for h in hooks.poststop.iter() {
-                container::execute_hook(logger, h, &oci_state).await?;
-            }
+            info!(&logger, "Poststop Hooks");
```

Comment:
It's better to refactor these lines using `if let Some()` like https://github.com/kata-containers/kata-containers/pull/5858/files#diff-5bf1516db5499202e6844836d7ebe31e26e2ded8ceec5536618409873a7fa3a1R1159-R1162

Reply:
fixed.